test/Api.IntegrationTest/Controllers/ServiceAccountsControllerTests.cs

```diff
@@ -130,4 +130,39 @@ public async Task UpdateServiceAccount()
         Assert.NotEqual(initialServiceAccount.RevisionDate, updatedServiceAccount.RevisionDate);
     }
 
+    [Fact]
+    public async Task CreateServiceAccountAccessToken()
+    {
+        if (_organization == null)
+        {
+            throw new ArgumentNullException(nameof(_organization));
+        }
```

Comment:
This check seems a bit odd?

Reply:
It had to do with how InitializeAsync works and not being a true constructor, so the editor gives you a warning that `_organization` might be null after exiting the constructor. 

I think for now, swapping back to non-nullable and removing those junk checks from the tests is the better play.